test/csuite/wt2323_join_visibility/main.c

```diff
@@ -249,13 +249,9 @@ thread_insert(void *arg)
          */
         key = (int)(__wt_random(&rnd) % N_RECORDS);
         maincur->set_key(maincur, key);
-/* FIXME-WT-6180: disable lower isolation levels. */
```

Comment:
From the discussion in WT-6180 lower level isolation levels are now read only. There's some talk in the ticket about leaving this code in place, however it's been sitting behind an `#if 0` for 2 years now and it doesn't look like that will change hence the removal. Happy to discuss this one further though.

Reply:
@quchenhao, would it be worth it to try what was in the `#if 0`?

Reply:
Had a look at the comments as well and it seems that it can be removed indeed.